POKEMON_OF_THE_DAY/PANDIUM.yaml

```diff
@@ -0,0 +1,41 @@
+version: 0.4
+base: node:16.14.2
```

Comment:
I don't understand how node versioning works and if this is the spot to dictate the version that its built with ... but I'm using `base: node:20.9.0` in whiplash<>gorgias, might be good to do that here too